roles/confluent.zookeeper/tasks/dynamic_groups.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Get Leader/Follower
+  shell: >
```

Comment:
whats the difference between `>` and `|` in yaml? ive been using the pipe everywhere

Reply:
https://yaml-multiline.info/ - I use it all the time! 😄

Reply:
ooo cool so with the `>` you dont need `\` with shell new lines.... interesting!